projects/components/src/common/subscription/subscription-tracker.ts

```diff
@@ -5,7 +5,7 @@
 
 import { OnDestroy } from '@angular/core';
 import { Observable, PartialObserver, Subscription } from 'rxjs';
-import { toSubscriber } from 'rxjs/util/toSubscriber';
+import { toSubscriber } from 'rxjs/internal-compatibility';
```

Comment:
Remove rx-js compat from our dependencies...

Reply:
Done in an existing commit.